README.md

```diff
@@ -8,14 +8,62 @@
   <a href="https://discord.gg/3G8cvWr7q4" ><img width="30%" src="https://user-images.githubusercontent.com/28778964/230801380-b8fb2ed3-fbcd-42ac-8e03-e45eb7c06e5e.png" /></a>
 </div>
 
+---
+<div align="center">
+
+![GitHub](https://img.shields.io/github/license/geeksesi/code-review-pals)
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/geeksesi/code-review-pals/.github/workflows/deploy.yml)
+![GitHub issues](https://img.shields.io/github/issues-raw/geeksesi/code-review-pals)
+![GitHub pull requests](https://img.shields.io/github/issues-pr-raw/geeksesi/code-review-pals)
+
+</div>
+
+---
 
 # Code Review Pals
 Code Review Pals is a community-driven platform for code review, designed to help developers learn from each other and improve their skills by providing a place to share their code and get feedback from peers.
 
 Our goal is to encourage developers of all levels to submit their code for review and to foster a supportive and collaborative environment where senior developers can help juniors learn and grow. We believe that code review is a vital part of the software development process, and that it helps improve code quality, catch bugs and security issues, and share knowledge.
 
-## Contribution
+
+```json
+"license": "MIT",
+  "stack": {
+    "php": "^8.1",
+    "laravel": "^10.0",
+    "react": "^18.2",
+    "npm": "^8.12"
+    }
+```
+
+## Installation
+
```

Comment:
we need a section for `.env`
about how configure login as github.
about redis, horizon queues, 

also it will be awesome if we mention about Sail, it's easy to use